integrated_channels/blackboard/migrations/0007_auto_20210909_1536.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.24 on 2021-09-09 15:36
+
```

Comment:
Why so many migration files for a single PR?

Reply:
oh, nvm, they are for different models.py files for different integrated channels

Reply:
yeah idk, when I ran migrate, it generated a new one for each model, is that even typical?

Reply:
I am guessing since they are in different folders?

Reply:
yeah since each subclass is in its own folder and migrations also reside in multiple folders